pre-docs/bolt_options.md

```diff
@@ -154,9 +154,10 @@ To have Bolt securely prompt for a password, use the `--password` or `-p` flag w
 
 ## Rerunning commands based on the last result
 
-After every execution, Bolt writes information about the result of that run to a
-`.rerun.json` file inside the Bolt project directory. That file can then be
-used to specify nodes for future commands.
+After every execution, Bolt writes information about the result of that run
+to a `.rerun.json` file inside the [Bolt project
+directory](./bolt_project_directory.md). That file can then be used to
```

Comment:
I can't put an inline link in a short description b/c DITA restrictions and SEO best practices. Will add a related information link to this section instead.